src/components/Book.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Book = () => (
+  <li>
+    <p>
+      Think and grow rich
+      <span>Napoleon Hill</span>
+      <button type="button">delete</button>
+    </p>
+
+  </li>
+);
+
+export default Book;
```

Comment:
- There are two issues here you need to work on.
- [ ] You need to use props here to pass and get the data from `BookList.js`. One of the purposes of this project is to learn how to use props. So, kindly use `props` here. I am giving an example below:
```
const Book = (props) => {
  const {
    title, author, category
  } = props;
return (
  <li>
    <div>
      <p>{title}</p>
      <p>{author}</p>
      <p>{category}</p>
    </div>
  </li>
);
};
```

- [ ] We use linters to avoid unwanted bugs and syntax errors. So by commenting linters, we try to skip this error, like you did here using `/* eslint-disable react/prop-types */`. But it is not a good choice. You can easily solve this issue by googling `how to fix prop-types linter error in react`. Kindly fix this issue.